File: src/main/java/com/paragon/client/systems/module/hud/impl/HUDEditor.java
package com.paragon.client.systems.module.hud.impl;

import com.paragon.api.module.Category;
import com.paragon.api.module.IgnoredByNotifications;
import com.paragon.api.module.Module;
import com.paragon.client.systems.module.hud.HUDEditorGUI;
import com.paragon.api.setting.Bind;
import org.lwjgl.input.Keyboard;

@IgnoredByNotifications
public class HUDEditor extends Module {

    public static HUDEditor INSTANCE;

    public HUDEditor() {
        super("HUDEditor", Category.HUD, "Lets you edit the HUD module positions", new Bind(Keyboard.KEY_P, Bind.Device.KEYBOARD));

        INSTANCE = this;
    }

    @Override
    public void onEnable() {
        mc.displayGuiScreen(new HUDEditorGUI());
        toggle();
    }
}

File: src/main/java/com/paragon/client/systems/module/hud/impl/HUDEditor.kt
package com.paragon.client.systems.module.hud.impl

import com.paragon.api.module.Category
import com.paragon.api.module.IgnoredByNotifications
import com.paragon.api.module.Module
import com.paragon.api.setting.Bind
import com.paragon.client.systems.module.hud.HUDEditorGUI
import org.lwjgl.input.Keyboard

/**
 * @author Surge
 */
@IgnoredByNotifications
object HUDEditor : Module("HUDEditor", Category.HUD, "Lets you edit the HUD module positions", Bind(Keyboard.KEY_P, Bind.Device.KEYBOARD)) {

    override fun onEnable() {
        minecraft.displayGuiScreen(HUDEditorGUI())
        toggle()
    }

}